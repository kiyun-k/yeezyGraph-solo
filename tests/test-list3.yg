int main() {
  List<int> i;
  i = new List<int>();
  i.l_add(3);
  i.printList();
  i.l_remove(0);
  i.printList();
  i.l_add(3);
  i.l_insert(0, 6);
  i.printList();
  print(i.isEmpty());
  print(i.size());
  i.removeAll();
  print(i.isEmpty());
  print(i.size());
  return 0;
}
