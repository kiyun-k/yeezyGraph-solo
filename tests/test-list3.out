{3}
{0}
{6, 3}
false
2
true
0